src/ValueObject/MediaValueObject.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme\ValueObject;
+
+/**
+ * Handle information about a media item.
+ */
+class MediaValueObject extends ValueObjectBase {
+
+  /**
+   * The name of the media.
+   *
+   * @var string
+   */
+  protected $name;
+
+  /**
+   * Media Source.
+   *
+   * @var string
+   */
+  protected $src;
+
+  /**
+   * MediaType constructor.
+   *
+   * @param string $src
+   *   Media URL, including Drupal schema if internal.
+   * @param string $name
+   *   Name of the media, e.g. "example.xxx".
+   */
+  public function __construct(string $src, string $name = '') {
```

Comment:
Constructors on value objects should be set to `private`.

Reply:
see https://github.com/openeuropa/oe_theme/pull/157#discussion_r260776257